vars/buildFailureMessage.groovy

```diff
@@ -0,0 +1,20 @@
+import com.cloudbees.groovy.cps.NonCPS
+import org.apache.commons.io.IOUtils
+@NonCPS
+def call(){
+    String ERROR_STRING = "ERROR"
+    List<String> message = []
+    Reader performance_log = currentBuild.getRawBuild().getLogReader()
+    String logContent = IOUtils.toString(performance_log)
+    performance_log.close()
+    performance_log = null
```

Comment:
Nit: Why do we reassign it to null?

Reply:
Stream object will be eligible for garbage collection